admin_app/src/app/dashboard/components/overview/HeatMap.tsx

```diff
@@ -25,8 +26,8 @@ const HeatMap = ({ data }: { data: any }) => {
       floating: false,
       style: {
         fontSize: "16px",
-        fontWeight: "bold",
-        fontFamily: undefined,
+        fontWeight: 500,
```

Comment:
Would have used semibold but it wasn't working within Apex chart! But using a number seems to work.